common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -1005,6 +1005,11 @@
           "The input of <functionName> can't be <dataType> type data."
         ]
       },
+      "UNSUPPORTED_MODE_DATA_TYPE" : {
+        "message" : [
+          "The <mode> does not support the <child> data type, because there is MapType with collated fields."
```

Comment:
PySpark and SQL users might be not aware of Scala/Java types like `MapType`. Let's follow common convention, and SQL types.

Reply:
@MaxGekk what do you think of the above?